ios/Podfile.lock

```diff
@@ -65,7 +65,7 @@ PODS:
   - jail-monkey (2.3.2):
     - React
   - libevent (2.1.12)
-  - Mixpanel (3.6.1)
+  - Mixpanel (3.6.5)
```

Comment:
I guess it could cause some issues (found time ago)
Could you restore the current version?

Reply:
Sure! Sorry I didn't notice it when runned `pod install`.
Restored in [4250e6c](https://github.com/pagopa/io-app/pull/3528/commits/4250e6c2ef7c00f45a46b5df51e676bb29e3cf82)